### DISCUSSION TITLE
How to pronounce KEDA

### DISCUSSION THREAD

**Discussion by laufermi** [kedacore/keda#2892] [General] [answered]
Hi angels, Our team already trial the KEDA in our Kubernetes cluster, but first: how to pronounce KEDA?

**Comment by Ritikaa96** [comment, marked as answer]
Hi @laufermi , Welcome to KEDA community, KEDA is actually **Kubernetes Event-driven Autoscaling** in abbreviated form. 
You can pronounce it as "kay.daa".

**Reply by zroubalik** [reply]
Yeah :) I know some people pronounce it as `kay.dah`, though I prefer `keh.dah`. Both versions are fine, we don't have any "official" pronounciation 😄 

Glad to hear that you are trying KEDA! If you are happy with it, would be nice to have you listed as a user: https://keda.sh/community/#users

**Reply by tomkerkhove** [reply]
We should add an agreed approach to https://github.com/kedacore/governance/blob/main/BRANDING.md.

+1 on kay.daa

**Comment by BryanDollery** [comment]
Key-dah

**Comment by frachel** [comment]
I say `ked-uh`

**Comment by murtuzamvista** [comment]
keeda
